src/RazorSdk/SourceGenerators/RazorSourceGenerationContext.cs

```diff
@@ -4,6 +4,8 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
```

Comment:
No thank you.

Reply:
Oops yeah, we don't actually need this. I was playing around with using `Regex.Escape` but decided against it.